.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - "4.2"
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
+  - npm install -g selenium-standalone
+  - selenium-standalone install
+  - selenium-standalone start > /dev/null 2>&1 &
```

Comment:
I should reinforce and clarify why I want to use something based on `npm run`. I consider global node modules to be an antipattern unless they are part of a developers workflow (e.g. `nodemon`).

For example, different projects might require a Selenium server but are using different versions of the same module. To avoid issues, it's best to stick to packages managed within `node_modules/`.

We act on this by setting them up as `devDependencies` and then using `npm run` to add `node_modules/.bin` to our `PATH`:

https://docs.npmjs.com/cli/run-script